charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -310,13 +325,28 @@ spec:
             value: {{ printf "%s" (join " " .Values.cassandra.metric_filters) }}
           {{- end }}
         {{- end }}
-       {{- if .Values.medusa.enabled }}
+
         volumeMounts:
+          {{- if .Values.medusa.enabled }}
           - name: cassandra-config
             mountPath: /etc/cassandra
           - name: podinfo
             mountPath: /etc/podinfo
-        {{- end }}
+          {{- end }}
+          {{- if .Values.cassandra.encryption.keystoreSecret }}
```

Comment:
Do these belong to this PR? The title/description talks about cassandra.yaml only.

Reply:
Yeah, these are intended so that we can give users a way configure internode and client encryption. I added the following properties values.yaml:

```yaml
# -- Specifies properties for configuring internode and client encryption. Specifically,
  # secrets are specified for which volumes and volume mounts are created. Internode
  # and client encryption should otherwise be configured through the cassandraYamlConfigMap
  # property.
  encryption:
    # -- A secret that contains the keystore.
    keystoreSecret:
    # -- Path in the cassandra container at which the keystore secret should be mounted.
    keystoreMountPath:
    # -- A secret that contains the truststore.
    truststoreSecret:
    # truststoreKey:
    # -- Path in the cassandra container at which the truststore secret should be mounted.
    truststoreMountPath:
```
You would need to configure your cassandra.yaml accordingly.